packages/setuptools/spec

```diff
@@ -0,0 +1,11 @@
+---
+name: setuptools
+
+dependencies:
+- zlib
+- bzip2
+- openssl
+- python27
+
+files:
+- setuptools/setuptools-36.0.1.tar.gz   # https://github.com/pypa/setuptools/archive/v36.2.7.tar.gz
```

Comment:
Not that it matters, but presume the link in the comment is wrong.

Reply:
Good spot, clearly yes.